src/librustc_mir/build/matches/simplify.rs

```diff
@@ -115,14 +116,14 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
                     ty::Int(ity) => {
                         // FIXME(49937): refactor these bit manipulations into interpret.
```

Comment:
These `FIXME`s look resolved now.

Reply:
Thank you for reporting. I forgot to remove this comment out.

Reply:
I think `let bias = 1u128 << (size.bits() - 1);` is not resolved.